Source/Core/DiscIO/VolumeGC.cpp

```diff
@@ -193,8 +193,14 @@ void CVolumeGC::LoadBannerFile() const
 
   GCBanner banner_file;
   std::unique_ptr<IFileSystem> file_system(CreateFileSystem(this));
-  size_t file_size = (size_t)file_system->GetFileSize("opening.bnr");
+  if (!file_system)
+    return;
+
+  std::unique_ptr<IFileInfo> file_info = file_system->FindFileInfo("opening.bnr");
+  if (!file_info)
+    return;
 
+  size_t file_size = (size_t)file_info->GetSize();
```

Comment:
Yes, but are we assuming that all systems that run Dolphin will be at least 32-bit? (I would assume so since the amount of RAM we use when emulating requires about 30 bits to be addressed, but I thought I should check to make sure)